### ISSUE TITLE
[rush] Is there a way to customize tag name generated in rush publish

### ISSUE THREAD

**Issue by ziofat** [microsoft/rushstack#2654]
## Summary

`rush publish` using package name and version as tag name, which is fair enough. However, as we want to manage tags in a more visual way, the current separator `_` is not good for us to keep things organize.

If we use separator `/` between package name and version, we may have ability to keep all related tags in one directory for specific package.

I'm not saying this is a good way because slash is also used in package names, but maybe a config can control this behavior?

## Standard questions

Please answer these questions to help us investigate your issue more quickly:

| Question | Answer |
| -------- | -------- |
| `@microsoft/rush` globally installed version? | 5.42.4|
| `rushVersion` from rush.json? | 5.42.4 |
| `useWorkspaces` from rush.json? | false|
| Operating system? | Mac |
| Would you consider contributing a PR? | Yes |
| Node.js version (`node -v`)? | 12.19.0 |

**Comment by iclanton** [comment]
I don't believe that's currently possible, but it would probably be easy to put together. You want to take a look at doing that?

**Comment by ziofat** [comment]
I would be glad to look into this. But I might need a little help with this.

My plan is to make this option go to some config file in `common/config/rush`, but not sure which one. IMO maybe `version-policy.json`?
